### PR TITLE
chore: keep hadoop-hdfs-client and hadoop-common version consistent

### PR DESCRIPTION
**Pull request by chenlixuan**
refer to #456

### CODE REVIEW

File: hugegraph-loader/pom.xml [hunk 1]
```diff
@@ -307,7 +308,7 @@
         <dependency>
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-common</artifactId>
-            <version>3.3.1</version>
+            <version>${hdfs.version}</version>
```
Comment (simon824) [inline hugegraph-loader/pom.xml:311]: `hadoop.version` maybe better

**Review by simon824 — APPROVED** [review]

File: hugegraph-loader/pom.xml [hunk 2]
```diff
@@ -40,6 +40,7 @@
         <release.name>${project.artifactId}</release.name>
         <final.name>apache-${release.name}-incubating-${project.version}</final.name>
         <jackson.version>2.12.3</jackson.version>
+        <hadoop.version>3.3.1</hadoop.version>
```
Comment (imbajin) [inline hugegraph-loader/pom.xml:43]: Could check whether there are other similar Hadoop dependencies in the project at the same time. If there are multiple module dependencies, you can put the version attribute under the root (pom)

**Review by imbajin — APPROVED** [review]

**Review by imbajin — CHANGES_REQUESTED** [review]
Need update the dependency file to fix the ci error @simon824 we should add a **PR template** and tell the devs how to modify it 

image

**Review by imbajin — CHANGES_REQUESTED** [review]
Need update the dependency file to fix the ci error @simon824 we should add a **PR template** and tell the devs how to modify it 

image

**Comment by simon824** [comment]
hi @chenlixuan , in order to check the LICENSE conveniently, when we update the dependency, we need also update this file  at the same time (https://github.com/chenlixuan/incubator-hugegraph-toolchain/blob/master/hugegraph-dist/scripts/dependency/known-dependencies.txt #L92) and make the version of dependencies in the file consistent with pom.xml

File: hugegraph-loader/pom.xml (hunk 2 above)
Comment (chenlixuan) [reply hugegraph-loader/pom.xml:43]: I will do it later.

**Review by simon824 — APPROVED** [review]

File: hugegraph-loader/pom.xml (hunk 2 above)
Comment (simon824) [reply hugegraph-loader/pom.xml:43]: > I will do it later.

@chenlixuan We can do it in the next pr,  unify the dependencies versions of the entire project.

**Review by imbajin — APPROVED** [review]

**Pull request merged**